test/TestBase.php

```diff
@@ -44,15 +52,28 @@ protected function setUp()
      */
     public function getClient(array $responses = [])
     {
-        $mock = new MockHandler($responses);
-        $stack = HandlerStack::create($mock);
+        $this->mockHandler = new MockHandler($responses);
+        $stack = HandlerStack::create($this->mockHandler);
 
         return new Lift(
           $this->accountId,
           $this->siteId,
-          $this->authKey->getId(),
-          $this->authKey->getSecret(),
-          ['handler' => $stack, 'auth_middleware' => false]
+          $this->publicKey,
+          $this->secretKey,
+          ['handler' => $stack, 'auth_middleware' => $this->getDummyAuthMiddleware()]
         );
     }
+
+    /**
+     * Get a dummy auth middleware.
+     */
+    protected function getDummyAuthMiddleware()
+    {
+        return function (callable $handler) {
+            return function (RequestInterface $request, array $options) use ($handler) {
+                return $handler($request, $options);
+            };
+        };
+    }
```

Comment:
We set in a dummy auth middleware, which does nothing. It replaces the real auth middleware, so we don't have to further complicate other tests.